phoenix-core/src/main/java/org/apache/phoenix/coprocessor/BaseScannerRegionObserver.java

```diff
@@ -374,20 +380,23 @@ RegionScanner getWrappedScanner(final ObserverContext<RegionCoprocessorEnvironme
                 dataRegion, indexMaintainer, null, viewConstants, null, null, projector, ptr, useQualiferAsListIndex);
     }
 
-    @Override
-    public KeyValueScanner preStoreScannerOpen(final ObserverContext<RegionCoprocessorEnvironment> c,
-        final Store store, final Scan scan, final NavigableSet<byte[]> targetCols,
-        final KeyValueScanner s) throws IOException {
+    @Override public KeyValueScanner preStoreScannerOpen(
```

Comment:
not sure what the diff is here..seems like mainly whitespace changes, right? Can we revert this?

Reply:
Again, this is Phoenix code template formatting kicking in.